Google.Api.Generator.Rest.Tests/GoldenTestData/Google.Apis.Assuredworkloads.v1/Google.Apis.Assuredworkloads.v1.csproj

```diff
@@ -16,6 +16,12 @@
     <Description>
       Google APIs Client Library for working with Assuredworkloads v1.
 
+      API reference documentation for this package is available at:
+      https://googleapis.dev/dotnet/Google.Apis.Assuredworkloads.v1/latest/api
```

Comment:
This as is is 404. Either remove the `api` segment or add `\Google.Apis.Assuredworkloads.v1.html`